### PR TITLE
Refactor actions and have separate actions for each root reducer.

### PR DESCRIPTION
**Pull request by BhagyashreeBhoyar**
Refactor actions and have separate actions for each root reducer.

### CODE REVIEW

File: src/actions/allMembersDataActions.js [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+/**
+ * setRedirectValueAction action will call when admin is login
+ * or admin is logout
+ * @param {Boolean} redirect
+ * @return {{redirect: Boolean, type: String}}
+ */
+export const setRedirectValueAction = ({ redirect }) => ({
+  type: 'SET_REDIRECT_VALUE',
+  redirect,
+});
```
Comment (Nikmahajan) [inline src/actions/allMembersDataActions.js]: This action can be placed separately.

File: src/actions/allMembersDataActions.js [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+
+/**
+ * setAdminLoginStateAction action will call when admin is login
+ * or logout
+ * @param {Boolean} adminLoginState
+ * @return {{adminLoginState: Boolean, type: String}}
+ */
+export const setAdminLoginStateAction = ({ adminLoginState }) => ({
+  type: 'SET_ADMIN_LOGIN_STATE',
```
Comment (Nikmahajan) [inline src/actions/allMembersDataActions.js]: Same here

File: src/actions/allMembersDataActions.js [hunk 3]
```diff
@@ -0,0 +1,364 @@
...
+/**
+ * uploadOptInFileResultsSuccessAction action will call
+ * when upload members optIn file API respond is success
+ * @param {Object} response
+ * @return {{type: String, failRecordIds: Null}}
+ */
+export const uploadOptInFileResultsSuccessAction = response => ({
+  type: 'UPLOAD_OPT_IN_FILE_SUCCESS',
+  failRecordIds: response.failRecordIds,
+  idNotExist: response.idNotExist,
```
Comment (Nikmahajan) [inline src/actions/allMembersDataActions.js]: Use object destructuring and also initialize it with some default value.

File: src/actions/allMembersDataActions.js (hunk 1 above)
Comment (BhagyashreeBhoyar) [reply src/actions/allMembersDataActions.js]: Done.

File: src/actions/allMembersDataActions.js (hunk 3 above)
Comment (BhagyashreeBhoyar) [reply src/actions/allMembersDataActions.js]: Done.

File: src/actions/allMembersDataActions.js (hunk 2 above)
Comment (BhagyashreeBhoyar) [reply src/actions/allMembersDataActions.js]: Done.

**Pull request merged**